test_get_function_docstring test for nifake fails for master

python3 -m tox returns as follows for nifake tests 

```
build/helper/documentation_helper.py::build.helper.documentation_helper.as_rest_
table PASSED
build/helper/documentation_helper.py::test_get_function_rst PASSED
build/helper/documentation_helper.py::test_get_function_docstring FAILED
build/helper/documentation_helper.py::test_get_rst_header_snippet PASSED
build/helper/documentation_helper.py::test_get_documentation_for_node_docstring
PASSED
build/helper/metadata_add_all.py::test_add_all_metadata_simple PASSED
build/helper/metadata_merge_dicts.py::test_merge_dict_second_is_empty PASSED
build/helper/metadata_merge_dicts.py::test_merge_dict_simple PASSED
build/helper/metadata_merge_dicts.py::test_merge_dict_first_is_empty PASSED
build/helper/metadata_merge_dicts.py::test_merge_dict_key_exists PASSED
build/helper/metadata_merge_dicts.py::test_merge_dict_recurse PASSED
build/helper/metadata_merge_dicts.py::test_merge_dict_replace_in_list PASSED
build/helper/metadata_merge_dicts.py::test_merge_dict_replace_in_dict_and_list P
ASSED
build/helper/metadata_merge_dicts.py::test_merge_dict_with_regex PASSED

================================== FAILURES ===================================
_________________________ test_get_function_docstring _________________________

    def test_get_function_docstring():
        actual_function_docstring = get_function_docstring('GetTurtleID', config
, 0)
        expected_function_docstring = '''Returns the **ID** of selected Turtle T
ype.

    Note: The RAPHAEL Turtles dont have an ID.

    Args:
        turtle_type (int):Specifies the type of Turtle type
            wanted to choose.

            +-------------------------------+---+--------------+
            | NIFake_VAL_LEONARDO (default) | 0 | LEONARDO     |
            +-------------------------------+---+--------------+
            | NIFake_VAL_DONATELLO          | 1 | DONATELLO    |
            +-------------------------------+---+--------------+
            | NIFake_VAL_RAPHAEL            | 2 | RAPHAEL      |
            +-------------------------------+---+--------------+
            | NIFake_VAL_MICHELANGELO       | 3 | MICHELANGELO |
            +-------------------------------+---+--------------+

            Note: You wont be able to import RAPHAEL

    Returns:
        turtle_id (float):Returns the **ID** of selected turtle.''' # noqa
>       assert expected_function_docstring == actual_function_docstring
E       AssertionError: assert 'Returns the ...ected turtle.' == 'Returns the *.
..ected turtle.'
E           Returns the **ID** of selected Turtle Type.
E
E           Note: The RAPHAEL Turtles dont have an ID.
E
E           Args:
E         -     turtle_type (int):Specifies the type of Turtle type
E         +     turtle_type (int): Specifies the type of Turtle type...
E
E         ...Full output truncated (20 lines hidden), use '-vv' to show

build\helper\documentation_helper.py:636: AssertionError
===================== 1 failed, 13 passed in 0.99 seconds =====================
```

--- a/build/helper/documentation_helper.py
+++ b/build/helper/documentation_helper.py
@@ -613,7 +613,7 @@ def test_get_function_docstring():
 Note: The RAPHAEL Turtles dont have an ID.
 
 Args:
-    turtle_type (int):Specifies the type of Turtle type
+    turtle_type (int): Specifies the type of Turtle type
         wanted to choose.
 
         +-------------------------------+---+--------------+
@@ -629,7 +629,7 @@ Args:
         Note: You wont be able to import RAPHAEL
 
 Returns:
-    turtle_id (float):Returns the **ID** of selected turtle.''' # noqa
+    turtle_id (float): Returns the **ID** of selected turtle.''' # noqa
     assert expected_function_docstring == actual_function_docstring
 
 

